### PR TITLE
Conversions tutorial

### PR DESCRIPTION
**Pull request by jsani-radar**
Part 2 of https://github.com/radarlabs/documentation/pull/278

Amend the existing tutorial to reflect the new conversions API and dashboard page

### CODE REVIEW

File: docs/tutorials/logging-and-analyzing-where-conversions-occur.md [hunk 1]
```diff
@@ -119,25 +123,33 @@ Radar.sendEvent(customType: "product_search", metadata: ["section":"furniture"])
...
-### Step 6: Use the Radar dashboard to view custom events
-You can see when and where custom events were generated directly in the Radar dashboard. On the [Events](https://radar.com/dashboard/events) page, watch as custom events start to stream in:
+### Step 6: Conversions overview in the Radar dashboard
+The [Report](https://radar.com/dashboard/report&view=conversions) page of the Radar dashboard has a _Conversions_ tab that provides an overview of all conversions logged for a project. Understand how engagement with your app differs between opted-in and opted-out users. See the total revenue across all conversions that have been logged with the `revenue` parameter.
+
+The below is an example of a fully populated _Conversions_ page with engagement metrics and logged conversions with and without a `revenue` amount:
+![Conversions report page](/img/tutorials/conversions-report-page.png)
+
+
+### Step 7: Analyze conversions across different dimensions
```
Comment (david-goodfellow) [inline docs/tutorials/logging-and-analyzing-where-conversions-occur.md:145]: `different location context dimensions`? Seems a bit unclear from the title if we would be slicing off of the dimensions they provide as parameters vs context Radar enriches

File: docs/tutorials/logging-and-analyzing-where-conversions-occur.md [hunk 2]
```diff
@@ -119,25 +123,33 @@ Radar.sendEvent(customType: "product_search", metadata: ["section":"furniture"])
...
+You can see when and where conversions were generated directly in the Radar dashboard. On the [Events](https://radar.com/dashboard/events) page, watch as conversions start to stream in:
 
-![Custom events dashboard](/img/tutorials/custom-events-dashboard.png)
+![Conversion events feed](/img/tutorials/conversion-events-feed.png)
 
-Click the _View_ icon on the right to view a custom event's details:
-![Custom events detail page](/img/tutorials/custom-events-detail-page.gif)
+Click the _View_ icon on the right to view a conversion's details:
+![Conversion event detail page](/img/tutorials/conversion-event-detail-page.gif)
```
Comment (david-goodfellow) [inline docs/tutorials/logging-and-analyzing-where-conversions-occur.md:151]: The detail page here is from December and outdated. Now we overlay the location context information (e.g. place geometry or geofence geometry on the geofence detail page). That would probably be more effective in being clear how we enrich their conversion events:
image

File: docs/tutorials.mdx [hunk 3]
```diff
@@ -15,7 +15,7 @@ Get to `Hello, world!` faster with tutorials and quickstarts! Or, get started wi
...
 - **[Building a store locator on iOS](/tutorials/building-a-store-locator-on-ios)**: Use the [iOS SDK](/sdk/ios), [forward geocode API](/api#forward-geocode), and [geofence search API](/api#search-geofences) to build a store locator with a map.
 - **[Localizing a website based on current country or city](/tutorials/localizing-a-website-based-on-current-country-or-city)**: Use the [web SDK](/sdk/web) and [IP geocoding API](/api#ip-geocode) to localize a website based on current country or city.
-- **[Analyzing where key customer interactions occur](/tutorials/analyzing-where-key-customer-interactions-occur)**: Send custom events using the [iOS](/sdk/ios) and [Android](/sdk/android) SDKs to analyze the location context of key customer interactions.
+- **[Logging and analyzing where conversions occur](/tutorials/logging-and-analyzing-where-conversions-occur)**: Send custom events using the [iOS](/sdk/ios) and [Android](/sdk/android) SDKs to analyze the location context of key customer interactions.
```
[the remark this answers is not in the document]
Comment (jsani-radar) [reply docs/tutorials.mdx:18]: good catch

[the remark this answers is not in the document]
Comment (jsani-radar) [reply docs/tutorials.mdx:18]: good catch

File: docs/tutorials/logging-and-analyzing-where-conversions-occur.md [hunk 4]
```diff
@@ -80,25 +80,25 @@ When your app starts, in application `onCreate()`, initialize the SDK with your
...
+To take advantage of Radar's conversions API, you need to identify the events that are important to your business. Here are some examples:
 
 | Event name | Example purpose                                                                                                     | Placement                           | Event properties                                                               |
 |------------|---------------------------------------------------------------------------------------------------------------------|-------------------------------------|--------------------------------------------------------------------------------|
-| `app_open`  | Understand where users are engaging with the app to prioritize personalized features                       | After app load completes      | NA                                                                             |
+| `purchase`   | Measure the revenue driven from in store mode and understand the distance from the store at the time of purchase | After a user makes a purchase              | `revenue` (number) <br /> `mode` (string) <br /> `in_store_mode` (boolean) |
```
[the remark this answers is not in the document]
Comment (jsani-radar) [reply docs/tutorials/logging-and-analyzing-where-conversions-occur.md:87]: right good call, atm you'd separately have to compute the distance between the conversion event + destination geofence

File: docs/tutorials/logging-and-analyzing-where-conversions-occur.md (hunk 1 above)
Comment (jsani-radar) [reply docs/tutorials/logging-and-analyzing-where-conversions-occur.md:145]: right, I was cribbing off the language in the body of this header - "view all of your conversions sliced by different dimensions, such as place chain or geofence tag"

think I'll go with "Analyze conversions across location context types"

**Pull request merged**